CHANGELOG.md

```diff
@@ -24,7 +24,8 @@ Unreleased: changes on master, not yet released
  - [#1992][] Matlab tests must now be explicitly listed in CMakeLists.
  - [#1970][] The `drake` pod Makefile now includes the `install` action.
  - [#1953][] Replace `valuecheckMatrix()` with `CompareMatrices()`.
- 
+ - [#2018][] Fix capitalization of Constrant and OptimizationProblem APIs to match style guide.
```

Comment:
So far in the life of this changelog, we have been fairly pedantic in backticking names that are code.  I am okay if we want to stop doing that, but we should be consistent.